### PR TITLE
Percentage implementation

### PR DESCRIPTION
**Pull request by denisvieira-dev**

### CODE REVIEW

File: src/Nox.Types/Types/Percentage/Percentage.cs [hunk 1]
```diff
@@ -1,9 +1,39 @@
...
+        {
+            result.Errors.Add(new ValidationFailure(nameof(Value), $"Could not create a Nox Percentage type a value {Value} is greater than than the maximum specified value of {_percentageTypeOptions.MaxValue}"));
+        }
+
+        Value = (float)Math.Round(Value, _percentageTypeOptions.Digits);
+
+        return result;
+    }
+
+    public override string ToString()
```
Comment (rochar) [inline src/Nox.Types/Types/Percentage/Percentage.cs:60]: he ToString must be culture independent, see the Lenght and the Lenght Tests on ToString and also our Readme.md
Thks

File: src/Nox.Types/Types/Percentage/Percentage.cs
Comment (rochar) [inline src/Nox.Types/Types/Percentage/Percentage.cs:1]: maybe I missing something,

I'm not seeing the Constructor and the From creation with Validation?

File: src/Nox.Types.EntityFramework/Types/Percentage/PercentageConverter.cs
Comment (rochar) [inline src/Nox.Types.EntityFramework/Types/Percentage/PercentageConverter.cs:1]: we need to add this converter and type to the EntityFramework Tests

**Review by rochar — CHANGES_REQUESTED** [review]

**Review by rochar — APPROVED** [review]
Looks good

**Review by gjorgji-grgovski-authoritypartners — APPROVED** [review]

**Pull request merged**